### ISSUE TITLE
AdwAboutWindow on GTK4 apps proportions are wrong [with screenshots]

### ISSUE THREAD

**Issue by Scotty-Trees** [ubuntu/yaru#3741]
**Expected Behavior**

The recently updated AdwAboutWindow, aka GTK4/libadwaita's new updated About Window dialog has been coming out for newly updated Gnome apps like the v43 of Extensions app (flatpak) and various other projects like Black Box Terminal (also a flatpak). Gnome 43 is out and I know Ubuntu 22.10 will support most, if not all, of Gnome 43. With flatpaks specifically some of those elements are already visible. 

**Actual Behavior**

In regards to the Yaru theme, I believe it might need an update to fix the AdwAboutWindow dialog issue, in which it shrinks the window into too small of dimensions that make it a lot harder for readability. I am running Ubuntu 22.04.1 LTS, so this may already be fixed in a 22.10 update, so if that is the case please let me know. 

The current Yaru theme on GTK4/libadwaita flatpak apps look like this: 

![Screenshot from 2022-09-26 11-53-47](https://user-images.githubusercontent.com/45135187/192325930-b2aba1f0-29de-4881-82e7-c24ebfce1321.png)

If I remove the Yaru flatpak theme via the --reset command by running `sudo flatpak override --reset` and let it default to Adwaita, the AdwAboutWindow takes on the correct size and dimensions as shown below here:

![Screenshot from 2022-09-26 11-55-19](https://user-images.githubusercontent.com/45135187/192326146-db34c56f-4a03-45e8-bdc9-93e5636485bb.png)

**Steps to Reproduce the Problem** --for the record, this is only tested on flatpak apps--

1. Download the flatpak app Extensions from https://flathub.org/apps/details/org.gnome.Extensions and make sure flatpak update is run to make sure you're using the latest v43 of this Extensions app.
2. Once installed, open Extensions app and few the About Window dialog
3. Check to see if you're using either the Adwaita theme for GTK4/libadwaita apps or using Yaru theme.

As a side note that is part of this issue, IF you force Flatpaks (GTK3 & GTK4) to respect your GTK theme:
Copy your GTK theme from Other Locations/Computer/usr/share/themes and create .themes folder in your Home directory. Copy the theme folder to here. Then give access to themes via flatpak cmd:
`sudo flatpak override --filesystem=$HOME/.themes`
`sudo flatpak override --env=GTK_THEME=Yaru-dark`

To undo these changes to default back to normal Adwaita theme: `sudo flatpak override --reset`

**Software that presents the issue**

Any flatpak app that uses the new AdwAboutWindow, will display the issue, while forcing flatpaks to using the Yaru theme as mentioned above.

**Yaru version**

```
scott@ubuntu:~$ apt show yaru-theme-gtk
Package: yaru-theme-gtk
Version: 22.04.4
Priority: optional
Section: misc
Source: yaru-theme
Origin: Ubuntu
Maintainer: Ubuntu Desktop Team <ubuntu-desktop@lists.ubuntu.com>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 33.9 MB
Depends: session-migration, gnome-themes-extra, gtk2-engines-pixbuf, gtk2-engines-murrine, libglib2.0-bin
Homepage: https://community.ubuntu.com/c/desktop/theme-refresh
Task: ubuntu-desktop-minimal, ubuntu-desktop, ubuntu-mate-core, ubuntu-mate-desktop
Download-Size: 630 kB
APT-Manual-Installed: no
APT-Sources: http://us.archive.ubuntu.com/ubuntu jammy/main amd64 Packages
Description: Yaru GTK theme from the Ubuntu Community
 This is the theme, better than a burrito, that is shaped by the community
 on the Ubuntu discourse.
 .
 This package contains the GTK+ 2 and 3 theme parts
 .
 If you want to follow development, more information here:
 https://community.ubuntu.com/c/desktop/theme-refresh.
```

**Screenshots**

Here's another example of how Black Box terminal looks under the Yaru flatpak theme being forced upon it:
![Screenshot from 2022-09-24 23-32-42](https://user-images.githubusercontent.com/45135187/192328660-028a66c2-5f6c-4098-8ed4-46057c07f8e6.png)

And here is how Black Box terminal looks with just the Adwaita theme only: 
![Screenshot from 2022-09-25 02-26-09](https://user-images.githubusercontent.com/45135187/192328841-228ceac4-d9f3-421d-9cda-fd59dca008fb.png)

**Upstream check**
It appears from my own testing that it does not happen with the Adwaita flatpak theme, only with the Yaru theme.

**Comment by Jupi007** [comment]
Hi @Scotty-Trees and thanks a lot for your very complete issue message :)
Sadly, this is really not fixable. I explain myself:
All libadwaita apps which will be installed (deb or snap) in next ubuntu version will not use the Yaru theme at all. Libadwaita doesn't allow this (sadly). The only themed part will be the accent color, which will follow user selected one thanks to a tweak.
We had previously based our Gtk4 theme on libadwaita built-in theme, but it makes no more sense, because it could even create issues in pure Gtk4 apps. So we reverted our theme on the vanilla one.

About your problem, we stopped to sync our theme with libadwaita changes before they add this dialog to the stylesheet, so it is normal that it looks bugged. And also, you are using the 22.04 yaru version which is even more outdated.

Btw, it is weird that you succeeded in theme those libadwaita apps!

In short: we would love to theme them, but we can't, and you forced this not allowed behaviour :D

I close this issue, sorry again.
If you have more question feel free to comment :)
